Procfile.dev

```diff
@@ -1,4 +1,4 @@
-web: env PORT=3000 bundle exec puma -C config/puma.rb
+web: env PORT=3000 RAILS_ENV=development bundle exec puma -C config/puma.rb
```

Comment:
Is this not also needed for sidekiq below?

Reply:
In the case of sidekiq, the error did not occur even if it was not explicitly set.
Apparently RAILS_ENV is set to development by default.

Reply:
I also set it to sidekiq to avoid confusion.